app/templates/audio_settings.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base.html" %} {% block content %}
+<div class="container mt-4">
+    <h1>Audio Settings</h1>
+    <form action="{{ url_for('get_choices') }}" style="width: 200px; margin: autio" method="post" id="form1">
+        <div class="from-check">
+            <label>Music:</label>
+            <input type="radio" id="Music On" name="music_on" value="True" class="form-check-input" />
+            <label for="Music On" class="form-check-label">On</label>
+            <input type="radio" id="Music Off" name="music_on" value="False" checked class="form-check-input" />
+            <label for="Music Off" class="form-check-label">Off</label>
+        </div>
+        <br>
+        <label for="music">Select music tracks:</label> <br> {% for song_name in song_names %}
+        <div class="from-check">
+            <input type="checkbox" class="music form-check-input" name="music_choices" value="{{song_name}}.mp3" disabled>
+            <label for="music" class="form-check-label">{{song_name}}</label><br> {% endfor %}
+        </div>
+        <br>
+        <label>Volume:</label>
+        <input id="rangeInput music" type="range" min="0" max="100" name="music_vol" disabled/>
+        <br><br>
+        <div class="from-check">
+            <label>Sound Effects:</label>
+            <input type="radio" id="Sound Effects On" name="sfx_on" value="True" class="form-check-input" />
+            <label for="Sound Effects On" class="form-check-label">On</label>
+            <input type="radio" id="Sound Effects Off" name="sfx_on" value="False" checked class="form-check-input" />
+            <label for="Sound Effects Off" class="form-check-label">Off</label>
+        </div>
+        <br>
+        <label for="Sound Effect">Select sound effect:</label> <br>
+        <select name="sfx_choice" style="width: 80px; height: 40px; margin-top: 5px;" id="sfx" disabled class="form-control"> 
+        {% for sfx_name in sfx_names %}
+        <option value="{{sfx_name}}.wav">{{sfx_name}}</option>
+        {% endfor %}
+        </select>
+        <br>
+        <label>Volume:</label>
+        <input id="rangeInput sfx" type="range" min="0" max="100" name="sfx_vol" disabled/>
+        <br /><br /><br />
```

Comment:
Use CSS

Reply:
Could you explain, please?
Did you mean to set the min and max values using css instead?

Reply:
Nevermind, I got it and fixed it.